conode/run_conode.sh

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env bash
```

Comment:
Now I see the light. True, but for the moment I have some hard-coded paths in there :( So first I'd need to change these... Is it worth it?